### PR TITLE
[QUARKS-89] Add LICENSE and NOTICE files to jars when building

### PR DESCRIPTION
**Pull request by Cazen**
I tested in my local and it works
Hmm, And I'm sorry but made lots of sync to apache github repo history..(Merge pull request 2, 3, 4 from apache/master). I tried to remove it, but failed. 
I think it does not affect to logic but please tell me if it cause any nconvenience

### CODE REVIEW

File: common-build.xml [hunk 1]
```diff
@@ -137,7 +137,9 @@
 	  </target>
 	
 	  <target name="jar" depends="compile">
-
+		<mkdir dir="${classes}/META-INF"/>
+		<copy file="${quarks.dir}/LICENSE" tofile="${classes}/META-INF/LICENSE.txt"/>
```
Comment (ddebrunner) [inline common-build.xml]: Just curious as to why the name change to the files (appending `.txt`)?

Comment (Cazen) [inline common-build.xml]: I'm sorry It's my mistake

**Comment by ddebrunner** [comment]
@Cazen 

The extra commits don't seem to be an issue but this is what I'm doing to keep my fork up to date:

* Don't use the ASF repo, only this mirror and your fork of this mirror. I.e. don't set any remotes pointing to the ASF repo.
* Never make changes to the master branch on your fork, always do any development on a branch, branches are cheap.
* Rebase your fork using this: https://github.com/IBMStreams/streamsx.topology/wiki/Useful-Git-GitHub-info which refers to http://blog.bigbinary.com/2013/09/13/how-to-keep-your-fork-uptodate.html

**Comment by Cazen** [comment]
I've carefully read your mention and it looks very helpful to me.
I'll apply it to my next work! Thank you @ddebrunner Have a good day!

**Pull request merged**